TerminalDocs/command-palette.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Windows Terminal Command Palette
+description: Learn how to use the command palette in the Windows Terminal.
+author: cinnamon-msft
+ms.author: cinnamon
+ms.date: 08/26/2020
+ms.topic: how-to 
+ms.service: terminal
+---
+
+# How to use the command palette in Windows Terminal ([Preview](https://aka.ms/terminal-preview/))
+
+The command palette lets you see which commands you can run inside Windows Terminal.
+
+## Invoking the command palette
+
+You can invoke the command palette by typing `ctrl+shift+p`. This can be customized by adding the `commandPalette` command to your key bindings.
+
+```json
+{ "command": "commandPalette", "keys": "ctrl+shift+p" }
+```
+
+## Command line mode
+
+If you'd like to enter a `wt` command into the command palette, you can do so by starting with the `>` character. This will run the `wt` command in the current window.
+
+## Defining commands
```

Comment:
At no point do we explain where commands come from in the file.

We probably need to add a paragraph that covers something like:

> Commands are automatically generated for each of the actions in your `keybindings`, when possible. If you'd like to add other commands to the command palette, you can do it like (this).

and make sure we specifically show that the commands are getting added to `keybindings`